tests/pyccel/test_pyccel.py

```diff
@@ -796,8 +796,7 @@ def test_basic_header():
 
 #------------------------------------------------------------------------------
 @pytest.mark.xdist_incompatible
-@pytest.mark.parametrize( "test_file", ["scripts/classes/generic_methods.py",
-                                        "scripts/classes/classes.py",
+@pytest.mark.parametrize( "test_file", ["scripts/classes/classes.py",
```

Comment:
Can you please explain this change?

Reply:
This change is my fault. There is some kind of undefined behaviour in `generic_methods.py`. See #1595.
The test often passes but not always. I moved the test to `test_interfaces_in_classes` in my PR. But I forgot to remove it from this list